packages/mock/src/mocks/searchparameters.json

```diff
@@ -187,14 +196,5 @@
     "base": ["Task"],
     "type": "token",
     "expression": "Task.identifier"
-  },
-  {
-    "resourceType": "SearchParameter",
```

Comment:
Looks like this `SearchParameter` is not in the list anymore?

Reply:
It looks like `Measure-url` was removed... It looks like someone hand edited the JSON file without updating the generator.  Would you mind adding `Measure-url` to `packages/generator/src/storybook.ts` as well?  🙏